docs/releasenotes.md

```diff
@@ -58,30 +58,30 @@ This release provides additional bug fixes and support for embedded build using
 - Added cmake flag for embedding system databases when building a static library
 - Added a way to disable building MIOpenDriver when building a static library
 - Added CC compiler detection in ROCm environment
-- Known issue: This release may show warnings for "obsolete configs" in the performance database. This can be fixed by rerunning tuning on a specfic network; [see tuning documentation](https://rocmsoftwareplatform.github.io/MIOpen/doc/html/perfdatabase.html#miopen-find-enforce)
+- Known issue: This release may show warnings for "obsolete configs" in the performance database. This can be fixed by rerunning tuning on a specfic network; [see tuning documentation](https://ROCm.github.io/MIOpen/doc/html/perfdatabase.html#miopen-find-enforce)
```

Comment:
The link is valid, but would it be better to point to the new site `https://rocm.docs.amd.com/projects/MIOpen/en/latest/perfdatabase.html#miopen-find-enforce`, as to ensure consistency between all documentation links?

Reply:
Thanks for this finding, fixed.

Reply:
[Notice] Actually it would be better to keep the link to the old documentation. Why: https://github.com/ROCm/MIOpen/pull/2597#discussion_r1424538544